<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDBusServerAddressSelector</class>
 <widget class="QFrame" name="CDBusServerAddressSelector">
  <property name="windowTitle">
   <string>DBus selector</string>
  </property>
  <layout class="QVBoxLayout" name="vl_DBusSelector">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QRadioButton" name="rb_DBusP2P">
     <property name="toolTip">
      <string>swift peer to peer setup, setup to run swift on 2 computers</string>
     </property>
     <property name="text">
      <string>DBus P2P server (1/2 computers)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fr_DBusServerAddress">
     <layout class="QGridLayout" name="gl_DBusServerAddressDetails">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_DBusServerPort">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>45000</string>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="cb_DBusServerAddress">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_DBusServerPort">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_DBusServerAddress">
        <property name="text">
         <string>Address:</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_DBusSession">
     <property name="text">
      <string>DBus session server</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_DBusSystem">
     <property name="text">
      <string>DBus system server</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rb_DBusP2P</tabstop>
  <tabstop>cb_DBusServerAddress</tabstop>
  <tabstop>le_DBusServerPort</tabstop>
  <tabstop>rb_DBusSession</tabstop>
  <tabstop>rb_DBusSystem</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
